src/foundation.cs

```diff
@@ -6092,6 +6913,25 @@ partial interface NSUrlSessionTask : NSCopying {
 		[Export ("priority")]
 		float Priority { get; set; } /* float, not CGFloat */
 #endif
+
+#if MONOMAC // Introspection fails for iOS
```

Comment:
A lot of selectors don't respond for `NSURLSession*` types but they do work when called. I suspect it's the same case for this property.

Look at monotouch-test and add a call to `Progress` just to see if it works (at runtime). If it does commit the removal of `#if MONOMAC`